speech/unit_tests/test_client.py

```diff
@@ -33,15 +26,26 @@ def _make_result(alternatives):
                 ) for alternative in alternatives
             ],
         )
+    else:
+        return cloud_speech_pb2.SpeechRecognitionResult(alternatives=[])
```

Comment:
Competing implementation. Rather than put the `return` in two branches, do the following at the beginning:

```python
if alternatives is None:
    alternatives = []
```

or even better just use an empty tuple (isn't mutable) as the default for `alternatives`